packages/calcite-components/src/components/switch/switch.scss

```diff
@@ -3,24 +3,22 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-switch-corner-radius: Specifies the corner radius of the component.
  * @prop --calcite-switch-handle-background-color: Specifies the background color of the component handle.
  * @prop --calcite-switch-handle-border-color: Specifies the border color of the component handle.
  * @prop --calcite-switch-track-background-color: Specifies the background color of the component track.
  * @prop --calcite-switch-track-border-color: Specifies the border color of the component track.
- * @prop --calcite-switch-corner-radius: Specifies the corner radius of the component.
- * @prop --calcite-switch-shadow: Specifies the shadow of the component.
+ * @prop --calcite-switch-handle-shadow: Specifies the shadow of the component handle.
+ * @prop --calcite-switch-handle-border-color-hover: Specifies the border color of the component handle when hovered.
+ * @prop --calcite-switch-shadow-hover: Specifies the shadow of the component handle when hovered.
+ * @prop --calcite-switch-handle-border-color-selected: Specifies the border color of the component handle when selected.
+ * @prop --calcite-switch-track-background-color-selected: Specifies the background color of the component track when selected.
```

Comment:
General q - should we use selected here or `checked` to match the prop?

Reply:
Good question. I've been going back and forth on `selected` (consistency) vs `checked` (correctness). I'll update it to `checked` per your suggestion.